verification/curator-service/ui/src/components/BulkCaseForm.tsx

```diff
@@ -231,6 +235,28 @@ class BulkCaseForm extends React.Component<
         return axios.put('/api/cases', c);
     }
 
+    /**
+     * Provides any validation errors associated with the provided cases.
+     *
+     * TODO: Find a way to parallelize these requests. We need the AxiosResponse
+     * value; so we can't use Promise.allSettled().
+     */
+    async validateCases(
+        cases: CompleteParsedCase[],
+    ): Promise<CaseValidationError[]> {
+        const validationErrors: CaseValidationError[] = [];
+        for (let i = 0; i < cases.length; i++) {
+            try {
+                await axios.post('/api/cases?validate_only=true', cases[i]);
+            } catch (e) {
+                validationErrors.push(
+                    new CaseValidationError(i + 1, e.response.data),
```

Comment:
Making sure I'm grokking correctly: I think it necessarily starts on the first row, since they're uploading a CSV. The row here is the row in the CSV, and doesn't capture anything like original row number or `caseReference.sourceEntryId`. Does this seem reasonable?